app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_reader :books, :people, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = Classroom.new('college')
+  end
+
+  # Method to list all books
+  def list_all_books()
+    puts 'All Books:'
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  # Method to list all people
+  def list_all_people()
+    puts 'All People:'
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
+
+  # Method to create a person
+  def create_person()
+    puts 'Enter person name:'
+    name = gets.chomp
+    puts 'Enter person age:'
+    age = gets.chomp.to_i
+    puts 'Is it a student? (y/n):'
+    is_student = gets.chomp.downcase == 'y'
+
+    if is_student
+      puts 'Enter student classroom:'
+      classroom = gets.chomp
+      person = Student.new(age, name, classroom, parent_permission: true)
+    else
+      person = Person.new(age, name)
+    end
+
+    people << person
+    puts 'Person created successfully.'
+  end
```

Comment:
- [ ] Great job so far @Naledi-Dikgale there are just a few changes you need to make and you'll be on your way to the next Milestone :wink: 
- [ ] Per the Project Requirements, You had to `Create a person (teacher or student, not a plain Person).` But I noticed that as seen in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), users aren't given the option to create a teacher or a student. However, when asked if `It is a student` by your app and I input `no`, the app throws an error and breaks. So please follow the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) and follow the project requirements so your app can work as expected without errors :100: 
- [ ] I have attached a video in the second comment below for your reference :+1: 
- [ ] After creating a student and using `option 2` to show the created student, I noticed that the student's details aren't being displayed as seen in the video so kindly fix that as well :+1: 

| Your UI | Given UI |
|--------|--------|
| ![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/102699958/c9cba243-d906-4ea6-8080-923344be6571) | ![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/102699958/dcfd6af9-0d62-4516-9def-723e8ea13d41) | 

- Project Requirements
![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/102699958/c2dd6ede-20a7-4648-998c-57179dcf51b1)